material/base.html

```diff
@@ -127,6 +127,16 @@
         <div class="md-main__inner">
           <div class="md-content">
             <article class="md-content__inner md-typeset">
+              <div class="admonition important" style="margin-top: 0">
+                <div class="admonition-title">These are the archived docs for Codacy
```

Comment:
I'm not comfortable with the "archived" term, because those docs versions are not really archived. I found some doc examples for other providers that archived docs are accessed through an "archive" link (eg. https://docs.gitlab.com/ee/). Also, SonarQube 8.2 is not accessible through their [docs page](https://docs.sonarqube.org/latest/)... How did you get there, @prcr? I can't find those docs anywhere... 😄 

Could we just refer to the version of the docs? WDYT, @nicklem @prcr ?

  ```suggestion
                <div class="admonition-title">These are the docs for Codacy
```

Reply:
This can also work well. I second Claudia's suggestion. @prcr thoughts?

Reply:
GitLab does have the banner mentioning "archived" docs even for their latest self-hosted version:

![image](https://user-images.githubusercontent.com/60105800/210540888-7e379ce9-062e-49d4-b930-27c1dc705eac.png)

However, I do agree this can be confusing. We're using "archived" more as a way to mean that the docs in that version are "frozen" and not being updated anymore.

If we don't mention the "archived" anymore, we can simplify even further and just have the title of the admonition like:

> This is the Codacy Self-hosted v9.0.0 documentation

For consistency, we can also go back to mentioning "documentation" instead of docs in the admonition body:

> For the latest updates and improvements, see the [latest Cloud documentation](https://docs.codacy.com/) instead.

How does this sound?

Reply:
Looking good to me. I updated the copy with the latest commit.